components/mpas-albany-landice/driver/glc_comp_mct.F

```diff
@@ -1410,7 +1410,8 @@ subroutine glc_import_mct(x2g_g, errorCode)
             n = n + 1
             sfcMassBal(i) = x2g_g % rAttr(index_x2g_Flgl_qice, n)
             floatingBasalMassBal(i) = x2g_g % rAttr(index_x2g_Fogx_qiceli, n)
-            ismip6_2dThermalForcing(i) = x2g_g % rAttr(index_x2g_So_tf2d, n)
+            if (index_x2g_So_tf2d /= 0) &
```

Comment:
why not use the flds_tf logical here?

Reply:
`flds_tf` is currently only available as a local module variable in `driver-mct/shr/seq_flds_mod.F90`, so from what I can see, there is not an easy way to make it available to the glc driver.

Reply:
The logical ocn_c2_glctf is placed in the infodata buffer which is available to all models.  Is that true/false independently of flds_tf ?

Reply:
I think that works -- I can test it quickly

Reply:
That does work and makes more sense than checking for a non-zero index. I also added it to the ocn driver and pushed to this branch

Reply:
@rljacob -- are you OK with this implementation?

Reply:
yes